src/inmanta/server/config.py

```diff
@@ -268,9 +268,9 @@ def validate_fact_renew(value: object) -> int:
 # Dashboard
 #############################
 
-dash_realm = Option("dashboard", "realm", "inmanta", "The realm to use for keycloak authentication.", is_str)
-dash_auth_url = Option("dashboard", "auth_url", None, "The auth url of the keycloak server to use.", is_str)
-dash_client_id = Option("dashboard", "client_id", None, "The client id configured in keycloak for this application.", is_str)
+dash_realm = Option("web-ui", "realm", "inmanta", "The realm to use for keycloak authentication.", is_str)
+dash_auth_url = Option("web-ui", "auth_url", None, "The auth url of the keycloak server to use.", is_str)
+dash_client_id = Option("web-ui", "client_id", None, "The client id configured in keycloak for this application.", is_str)
```

Comment:
I think we want to define all `web-ui` configuration options in the `inmanta-ui` extension as this extension primarily exist for web-interface related things.